tokio/Cargo.toml

```diff
@@ -9,6 +9,7 @@ name = "tokio"
 # - Create "v1.0.x" git tag.
 version = "1.14.0"
 edition = "2018"
+rust-version = "1.46"
```

Comment:
According to [this page](https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field), this emits a warning on versions older than 1.56.

Reply:
I don't think this is a problem for the users, because warnings from dependencies will not be displayed. (e.g., [syn has been using this field since 1.0.78](https://github.com/dtolnay/syn/commit/a1096c2fbe400e5762802e4737e7c4565334e52a), but if we build a crate that  has syn as a dependency using an older compiler, we will not see the warning.)

Reply:
Okay. It seems like we don't get any failures in CI due to this warning either.